linkerd/app/core/src/metrics.rs

```diff
@@ -102,9 +102,18 @@ pub struct OutboundEndpointLabels {
     pub server_id: tls::ConditionalClientTls,
     pub authority: Option<http::uri::Authority>,
     pub labels: Option<String>,
+    pub zone_locality: OutboundZoneLocality,
     pub target_addr: SocketAddr,
 }
 
+#[derive(Debug, Copy, Clone, Default, Hash, Eq, PartialEq, EncodeLabelValue)]
+pub enum OutboundZoneLocality {
+    #[default]
```

Comment:
TIL! this syntax stabilized in https://doc.rust-lang.org/std/default/trait.Default.html#enums in rust 1.62.0. :heart_eyes:

Reply:
It's super handy, it only works on field-less enum variants for now but I think there's plans to change that in the near future.